spec.md

```diff
@@ -142,6 +142,10 @@ The `<reference>` MUST NOT be in any other format. Throughout this document, `<n
 
 `[a-z0-9]+([._-][a-z0-9]+)*(/[a-z0-9]+([._-][a-z0-9]+)*)*`
 
+Throughout this document, `<reference>` as a tag name MUST be at most 128 characters in length and MUST match the following regular expression:
```

Comment:
I didn't see the term `tag name` before read right over it.

Please s/tag name/tag/ both here and above on line 140.